foundations/html_css/flexbox/flexbox-axes.md

```diff
@@ -21,9 +21,10 @@ The default direction for a flex container is horizontal, or `row`, <span id='fl
 ~~~
 
 ### Axes
-
 <span id='flex-axes'>No matter which direction you're using, you need to think of your flex-containers as having 2 axes: the main axis and the cross axis. It is the direction of these axes that changes when the `flex-direction` is changed. In _most_ circumstances, `flex-direction: row` puts the main axis horizontal (left-to-right), and `column` puts the main axis vertical (top-to-bottom).</span>
-
+<br>
+<img src="https://samanthaming.gumlet.io/flexbox30/4-flexbox-axes.jpg.gz" alt="main axes and cross axes" title="image Title" width="300" height="300"/>
```

Comment:
Looks like a good image. Could you put a space between the image and the text surrounding it on the bottom and top? They bleed a bit into each other and it would look a bit better with that separation.